library/core/src/char/convert.rs

```diff
@@ -193,6 +193,9 @@ impl From<u8> for char {
 }
 
 /// An error which can be returned when parsing a char.
+///
+/// This `struct` is created when using the
+/// [`char::from_str`](../../std/primitive.char.html#method.from_str) method.
```

Comment:
Please use intra-doc links so this doesn't break on re-exports.
```suggestion
/// [`char::from_str`] method.
```

Reply:
I don't even know why I didn't use it directly...